packages/yoastseo/src/languageProcessing/helpers/word/addWordboundary.js

```diff
@@ -26,7 +23,7 @@ export default function( matchString, positiveLookAhead = false, extraWordBounda
          * \u060C - Arabic comma
          * \u061B - Arabic semicolon
          */
-		wordBoundary = "[ \\u00a0\\u2014\\u06d4\\u061f\\u060C\\u061B\\n\\r\\t\.,\(\)”“〝〞〟‟„\"\+\\-;!¡\?¿:\/»«‹›" + extraWordBoundary + "<>";
+		wordBoundary = "[ \\u00a0\\u2014\\u06d4\\u061f\\u060C\\u061B\\n\\r\\t.,()”“〝〞〟‟„\"+\\-;!¡?¿:/»«‹›" + extraWordBoundary + "<>";
```

Comment:
I removed the useless escape. It looks like it's okay to do so. All specs are still passing